frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -22,6 +24,12 @@ const logout = () => {
     loading.value = true;
 };
 
+const showTermsAndConditions = () => {
+    toggleTermsCloseble();
+    showTerms();
+    profileSidebarState.toggleVisible();
```

Comment:
do we still have this "toggleVisible" method?

Reply:
the toggleVisible is for the ProfileSidebar itself, when the user click to see the terms and condition the profileSidebar will close.

Reply:
thanks I missed that one!